docs/content/getting-started/setup-openfga/configure-openfga.mdx

```diff
@@ -13,6 +13,8 @@ import TabItem from '@theme/TabItem';
 
 # Configuring OpenFGA
 
+Before configuring OpenFGA, follow [Getting Started](https://github.com/openfga/openfga?tab=readme-ov-file#getting-started) to set up and install openfga. You can then run `./openfga run` as a pre-compiled binary or using your preferred method.
```

Comment:
```suggestion
Refer to the [OpenFGA Getting Started](https://github.com/openfga/openfga?tab=readme-ov-file#getting-started) for info on the various ways to install OpenFGA.

The instructions below assume OpenFGA is installed and that you have the `openfga` binary in your PATH. If you have built `openfga` as a binary, but not in your path, you can refer to it directly (e.g. replace `openfga` in the instructions below with `./openfga` or `/path/to/openfga`).
```

Reply:
Where do we provide instructions on adding `openfga` to your path? It's not in the README I'm linking

Reply:
When you do brew install it does it automatically

Reply:
In that case, should I also update the readme so that these instructions are consistent?